### PR TITLE
Ftreguer restful api

### PR DESCRIPTION
**Pull request by ftreguer**

### CODE REVIEW

**Comment by dlecan** [comment]
Deploy preview for *front-sonarplanet* processing.

Built with commit b81b0aa9dd24feb0874edab0754064fe70636b8c

https://app.netlify.com/sites/front-sonarplanet/deploys/5af5a00a3672df1abfa5893e

File: README.md [hunk 1]
```diff
@@ -24,3 +24,6 @@ yarn integ
 ```
 yarn prod
 ```
```
Comment (dlecan) [inline README.md]: Missing `yarn test` documentation.

File: src/sonarplanet.ts [hunk 2]
```diff
@@ -1,60 +1,56 @@
...
+})
 
 let trackAddressNow = (event: Event) => {
-  event.preventDefault();
-  $('.alert').each((index: Number, alert: HTMLElement) => {
-    $(alert).css('display', 'none')
+  event.preventDefault()
+  resetAlerts()
+  var inputAddress = document.getElementById("trackAddress") as HTMLInputElement
+  SonarPlanetAPI.registerServiceWorker().then((serviceWorkerRegistration) => {
```
Comment (dlecan) [inline src/sonarplanet.ts]: Too many callbacks and nested code.
Let's try to reduce this code in the next sprint.

File: test/sonarplanet.test.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+import * as SonarPlanetAPI from "../src/sonarplanetAPI"
+
+
+describe('SonarPlanet API', () => {
+
+  it('getBrowserId returns a unique id, thx to sessionStorage', done => {
+    let idSet = new Set()
+    for (var i = 0; i < 4000; i++) {
+      idSet.add(SonarPlanetAPI.getBrowserId())
```
Comment (dlecan) [inline test/sonarplanet.test.ts]: You should also test the content of the (mocked) localStorage: empty at the beginning, with only one value at the end.

Comment (ftreguer) [reply test/sonarplanet.test.ts]: Good idea!

Comment (ftreguer) [reply test/sonarplanet.test.ts]: Done!

File: README.md (hunk 1 above)
Comment (ftreguer) [reply README.md]: Done!
